src/lib/mina_graphql/mina_graphql.ml

```diff
@@ -2948,6 +3097,15 @@ module Mutations = struct
               ~fee_token ~fee_payer_pk:from ~valid_until ~body ~signature
             |> Deferred.Result.map ~f:Types.UserCommand.mk_user_command)
 
+  let send_snapp =
```

Comment:
Don't we have a separate command for creating the snapp account vs sending the snapp? Is this covered by the `create_account` mutation?

Reply:
Not sure what you're asking. This new mutation sends a Snapp transaction; the existing `createAccount` mutation creates an account, and in the current branch, accounts have a `snapp` field.